client/src/visualiser-src/sorting-visualiser/animation-producer/SortsMergeAnimationProducer.ts

```diff
@@ -0,0 +1,67 @@
+import { mergeCodeSnippet } from "../util/codeSnippets";
+import { getX, getCx, getY } from '../util/helpers';
+import GraphicalSortsElement from "../data-structure/GraphicalSortsElement";
+import SortsAnimationProducer from "./SortsAnimationProducer";
+import { textCy, defaultColour, sortedColour, checkingColour } from "../util/constants";
+
+export default class SortsMergeAnimationProducer extends SortsAnimationProducer {
+    public renderMergeCode() {
+        this.renderCode(mergeCodeSnippet);
+    }
+
+    public highlightSorting(low: number, high: number, elemensList: GraphicalSortsElement[]) {
+        for (let i = low; i <= high; i += 1) {
+            this.addSequenceAnimation(elemensList[i].boxTarget.animate().attr({ fill: checkingColour }));
+            this.addSequenceAnimation(elemensList[i].numberTarget.animate().attr({ fill: checkingColour }));
+            this.addSequenceAnimation(elemensList[i].boxTarget.animate().attr({ stroke: checkingColour }));
+            this.addSequenceAnimation(elemensList[i].numberTarget.animate().attr({ stroke: checkingColour }));
+        }
+    }
+
+    public compareLowerandEqual(first: GraphicalSortsElement, second: GraphicalSortsElement): boolean {
+        // highlight color to green
```

Comment:
Not sure if this is true?

Reply:
yeah, you are correct. I attempted to change it to green initially but later found it better to be more transparent and forgot to change the comments.